INSTALL.md

```diff
@@ -1,143 +1,120 @@
-   - mpmath >= 0.19
-   - blosc2 >= 2.3.0
-   - future
-   - ndindex >= 1.4
-   - msgpack
-   - six >= 1.5
-2. Create conda environment/ virtual environment for the installation of PISA. Assume that the name of the environment is `pisa_env` you can choose your preferred version of python >= 3.10
+# Installation Guide
+
+
+## Instruction to install PISA using Anaconda or Miniconda
+
+The following commands are intended for execution in a terminal. If you encounter any problems with the installation, please create an issue and/or ask on the IceCube slack (if possible).
+
+**1.** Install the essential dependencies
+   
+   Choose either Anaconda (~4.4 GB) OR Miniconda (~480 MB)
+
+   **Anaconda** (https://docs.anaconda.com/anaconda/), e.g.
+   ```bash
+   mkdir -p PATH_TO_ANACONDA/anaconda3
+   wget https://repo.anaconda.com/archive/Anaconda3-INSTALLER_VERSION-Linux-x86_64.sh -O PATH_TO_ANACONDA/anaconda3/anaconda.sh
+   bash PATH_TO_ANACONDA/anaconda3/anaconda.sh -b -u -p PATH_TO_ANACONDA/anaconda3
+   rm PATH_TO_ANACONDA/anaconda3/anaconda.sh
+   ```
+   You can find a list of available INSTALLER_VERSIONs at https://repo.anaconda.com/archive/.
+   
+   **Miniconda** (https://docs.anaconda.com/miniconda/), e.g.
+   ```bash
+   mkdir -p PATH_TO_ANACONDA/miniconda3
+   wget https://repo.anaconda.com/miniconda/Miniconda3-INSTALLER_VERSION-Linux-x86_64.sh -O PATH_TO_ANACONDA/miniconda3/miniconda.sh
+   bash PATH_TO_ANACONDA/miniconda3/miniconda.sh -b -u -p PATH_TO_ANACONDA/miniconda3
+   rm PATH_TO_ANACONDA/miniconda3/miniconda.sh
+   ```
+   You can find a list of available INSTALLER_VERSIONs at https://repo.anaconda.com/miniconda/.
+   
+   **If you install on the cobalts, rather use /data/user/YOURNAME/ than $HOME/ for PATH_TO_ANACONDA**
+   
+   **Note** that you will also need git. It is already installed on the cobalts, ask your local system administrator if it is not on your local machine. The other non-python requirements are listed [here](https://github.com/icecube/pisa/blob/master/INSTALL.md#required-dependencies) and should already come with the conda environment.
+
+   Other required libraries will be installed automatically during the setup (listed in https://github.com/icecube/pisa/blob/master/setup.py). If something is missing, you can install it via pip afterwards.
+
+**2.** Create an environment for the installation of PISA (after activating anaconda). E.g. for miniconda with python 3.10 use:
+
+   ```bash
+   source PATH_TO_ANACONDA/miniconda3/bin/activate
+   conda create -n NAME_OF_YOUR_PISA_ENV python=3.10
+   ```
+
+**3.** Activate the newly created environment
+
+   ```bash
+   conda activate NAME_OF_YOUR_PISA_ENV
+   ```
+
+**4.** Clone the PISA repository from github (https://github.com/icecube/pisa.git). You can also create your own fork first. For more information on how to obtain the pisa source code see [obtain-pisa-sourcecode](https://github.com/icecube/pisa/blob/master/INSTALL.md#obtain-pisa-sourcecode)
```

Comment:
Replace "[obtain-pisa-sourcecode]" with "[below]" and use section link instead of URL (https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#section-links). Add period to end of sentence.